application/cms/models.py

```diff
@@ -537,9 +537,6 @@ def to_dict(self, with_dimensions=False):
             "need_to_know": self.need_to_know,
             "ethnicity_definition_summary": self.ethnicity_definition_summary,
             "related_publications": self.related_publications,
-            "contact_name": self.contact_name,
```

Comment:
Nice how this just excluded the secondary contact details.

And nice how the JSON export must have included them :man_facepalming: 

We should probably kill `to_dict` altogether.